plop-templates/Component/Component.tsx.hbs

```diff
@@ -3,10 +3,13 @@ import React from 'react';
 import styles from './{{pascalCase name}}.module.css';
 
 export interface Props {
+  // Component API
```

Comment:
Improve: Add newer templating where we group the props by those needed by eng (this area), from those to match what's in figma (Design API)